src/nonlinear/room-edits/sector-1/room-1A.s

```diff
@@ -0,0 +1,7 @@
+; Sector 1 - Ridley Arena Access
+; Slightly adjust exit-distance from door with gadora to prevent instant damage if the gadora is not defeated.
+
+.org Sector1Doors + 39h * DoorEntry_Size + DoorEntry_ExitDistanceX
+.area 1
+    .db     23h
```

Comment:
should we have a constant for this somewhere instead of copy pasting the value over and over?